riff-raff/app/conf/context.scala

```diff
@@ -240,7 +240,13 @@ class Config(configuration: TypesafeConfig, startTime: DateTime) extends Logging
     }
   }
 
-  def credentialsProviderChainV1(accessKey: Option[String] = None, secretKey: Option[String] = None): AWSCredentialsProviderChainV1 = {
+  /**
+    * Credentials for use with V1 of the AWS SDK.
+    * Considered legacy as, where possible, V2 of the AWS SDK should be used.
+    *
+    * @deprecated Use V2 credentials - see [[credentialsProviderChain]].
```

Comment:
Minor: the comment here is useful but I'm not convinced that a deprecation warning on the credentials provider is the best way to communicate that we're moving away from v1. IMO it will likely lead to noise when compiling. We might be better off with a Trello card for the upgrade?

Reply:
Hadn't considered this - looking at the build log for CI, there are _lots_ of deprecation warnings, and I agree this will easily be lost. Removed.